prompt.go

```diff
@@ -236,16 +236,24 @@ func (p *Prompt) Input() string {
 
 func (p *Prompt) readBuffer(bufCh chan []byte, stopCh chan struct{}) {
 	log.Printf("[INFO] readBuffer start")
+
+	bch := make(chan []byte)
+	go func() {
```

Comment:
Is there something benefit to generate another worker goroutine to execute `p.in.Read` ?